uuid/mod.ts

```diff
@@ -33,6 +33,11 @@ import { generate as generateV1, validate as validateV1 } from "./v1.ts";
 import { generate as generateV3, validate as validateV3 } from "./v3.ts";
 import { validate as validateV4 } from "./v4.ts";
 import { generate as generateV5, validate as validateV5 } from "./v5.ts";
+import {
+  extractTimestamp,
```

Comment:
a minor point for some consideration

I wonder if it's ok to export this name from here as all other top level methods have version suffixes.. What do you think about the name `extractTimestampV7`?

Reply:
This makes sense, I think. Other UUID types actually do store timestamps and we don't have functions to extract them right now.

Reply:
Wait sorry, I misread, we shouldn't do this. It's already on the v7 object that is being exported, we should do V7 on the import, in case in the future we decide to add more.